docs/config.md

```diff
@@ -73,6 +73,12 @@ model_names: <comma-delimited-string>
 # Generate summary of results
 [ summarize: <boolean>  | default: true ]
 
+# Number of top configs to show in summary plots
+[ configs_shown_per_model: <int> | default: 3]
+
+# Number of top configs to save across ALL models, none saved by default
+[ top_n_configs: <int> | default: 0 ]
```

Comment:
I think it would be better to be backward compatible. As far as I remember, `top_n_configs` used to have different functionality in the previous release. We can use the new parameter (i.e. `configs_per_model`) for the new functionality and keep the old one for the previous functionality.

Reply:
Technically, this is not yet a feature that we were supposed to use in any stable version, which was why I had hidden it in the autofill. I think the fact that we're just now adding it to the docs gives us some leeway in this regard? Im getting rid of `top_n_configs` altogether and just having `configs_shown_per_model`  and `top_n_models` which is by default across all models.